src/core/emailParser.ts

```diff
@@ -0,0 +1,36 @@
+const postalMime = require('postal-mime/dist/node').postalMime.default;
+import {EmailContent} from '../model/emailContent.model';
+
+
+export default class EmailParser extends postalMime {
+    emailContent: EmailContent;
+    constructor() {
+        super();
+    }
+
+    // Email in RFC 822 format.
+    async parse(message: string): Promise<EmailContent> {
+        // A message is invalid if it starts with a space.
+        message = message.trim();
+
+        try {
+            this.emailContent = await super.parse(message);
+            this.emailContent.subject = this.subject;
+            this.emailContent.html = this.html;
+            this.emailContent.attachments = this.emailContent.attachments;
+
+            return (this.emailContent);
+        } catch (err) {
+            throw err;
```

Comment:
I think doing this is equivalent to not using try/catch at all.

Reply:
To be clear - I'm not saying you should or should not catch errors here, but certainly this is just pointless, no?

Reply:
You are right, it's pointless. I forgot to remove it. I only used it to test and know the different types of errors from the email parser in the external JS file.